distribution/lib/Standard/Test_New/0.0.0-dev/src/Group.enso

```diff
@@ -9,9 +9,9 @@ type Group_Builder
     Impl builder
 
     ## Specifies a single test.
-    specify : Text -> (Nothing -> Any) -> Nothing
-    specify self (name:Text) ~code =
-        self.builder.append <| Spec.Impl name (_ -> code)
+    specify : Text -> (Nothing | Text) -> (Nothing -> Any) -> Nothing
+    specify self name ~code pending=Nothing =
```

Comment:
I guess the `pending` argument isn't specified in the signature at all. Maybe it is better to use ascribed inline types.

Reply:
Using inline ascribed types on the latest version